linux_os/guide/system/accounts/accounts-session/accounts_tmout/ansible/shared.yml

```diff
@@ -11,21 +11,37 @@
   {{% set system_configuration_using_etc_bashrc_expected = true %}}
 {{% endif %}}
 
+- name: Set tmout line fact
+  set_fact:
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)